src/export/ExtensionExporter.ts

```diff
@@ -1,15 +1,27 @@
 import { StructureDefinitionExporter } from './StructureDefinitionExporter';
 import { FSHTank } from '../import/FSHTank';
 import { StructureDefinition } from '../fhirtypes';
+import { FHIRDefinitions } from '../fhirdefs';
 
 export class ExtensionExporter extends StructureDefinitionExporter {
-  constructor() {
-    super();
+  constructor(FHIRDefs: FHIRDefinitions) {
+    super(FHIRDefs);
   }
 
-  // eslint-disable-next-line
+  /**
+   * Exports Extensions to StructureDefinitions
+   * @param {FSHTank} tank - The FSH tank we are exporting
+   * @returns {StructureDefinition[]}
+   */
   export(tank: FSHTank): StructureDefinition[] {
-    // TODO: Make meaningful export of StructureDefinitions from contents of tank
-    return [];
+    const structDefs: StructureDefinition[] = [];
+    for (const doc of tank.docs) {
+      for (const [, extension] of doc.extensions) {
+        // TODO: catch errors and log them once logging is in place
```

Comment:
So it's easier to integrate the logger later, I'd suggest you go ahead and wrap this in a try/catch now and put a `console.error(...)` in the catch.

Also, just so we're on the same page, I think the only error we should toss up to this level is when the parent is not found.  I think most other errors should be caught inside the `exportStructDef` function so that we don't bail on the whole structdef just because of one error...